src/vanilla/store.ts

```diff
@@ -100,6 +100,10 @@ type AtomState<Value = AnyValue> = {
   n: number
   /** Object to store mounted state of the atom. */
   m?: Mounted // only available if the atom is mounted
+  /** Listener to notify when the atom value is updated. */
+  u?: () => void
```

Comment:
I'd like to make a patch release with only dev store improvement. So, let's remove this in this PR.
```suggestion
```